onboarding-checklist.md

```diff
@@ -0,0 +1,145 @@
+# Onboarding Checklist #
+
+## Instructions ##
+
+When someone new joins the team:
+
+1. A team member adds the new team member to the `cisagov` organization in
+GitHub.
+1. Create a new issue called "Onboard [NewPerson]".
+1. View the raw source of this file.
+1. Copy the text after the line to the issue, replacing "NewPerson" with the
+new person's name and "Buddy" with the onboarding buddy's name.
+1. Remove any block of tasks that doesn't apply to the new team member's role
+1. Submit the issue.
+1. Assign the new person and the new person's Onboarding Buddy to the issue.
+1. Move the issue into _In Progress_.
+
+---
+
+To get NewPerson productively contributing to the team, Buddy should help
+NewPerson complete a prescribed set of tasks that will bring them up to speed.
+
+## Directions ##
+
+**NewPerson and Buddy:** Try to go through your checklists in order.
+
+**Buddy:** If you can’t complete any of the items on your checklist
+personally, _you are responsible for ensuring that someone with the correct_
+_access completes that item_.
+
+## New Person checklist ##
+
+### Getting to know VM Fusion ###
+
+- [ ] Take judicious notes on what about your onboarding process is confusing
+or frustrating.
+  - Please share this information with your buddy (or the team) so we can make
+  things better.
+  - If you notice a problem (especially with things like documentation), you
+  are more than welcome to fix it!
+  - You can also file issues and pull requests on the [template onboarding checklist]
+- [ ] Figure out who your onboarding buddy is (they should reach out to you).
+- [ ] Once you've finished the checklists below, make suggestions for steps
+that would have improved your onboarding experience as pull requests on the
+[template onboarding checklist] used to make this issue.
+
+### Required items for all team members ###
+
+Completing these items helps us fulfill security and compliance requirements.
+If you get stuck, or if these requirements are confusing, ask for help from
+your onboarding buddy.
+
+Pre-requisites:
+
+- [ ] Complete [PALMS Cybersecurity Training](https://etms.hq.dhs.gov),
+including Mandatory Cyber Security and Privacy Training, including accepting
+the Rules of Behavior, which is required before we can give you access to any
+systems, like Trio.
+- [ ] Go through the [FISMA-Ready Github guide](https://github.com/fisma-ready/github)
+to set up your Github account to be added to CISA
```

Comment:
This is a complete sentence and could end with a period.

Reply:
Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4

Reply:
Doesn't appear to be changed, which is OK if you don't want to.

Reply:
Fixed in a848650149665216a068543dd1b50962d7ec2737